### PR TITLE
Acs metadata

### PR DESCRIPTION
**Pull request by stuartlynn**
- Generates the Metadata asset for the ACS products

### CODE REVIEW

File: python/popgetter/metadata.py [hunk 1]
```diff
@@ -92,5 +60,37 @@ class MetricMetadata(BaseModel):
...
+    date_published: date = Field(description="The date on which the data was published")
+    reference_period: tuple[date, date | None] = Field(
+        description="The range of time for which the data can be assumed to be valid. Should be in the format (start_date, end_date)."
+        " If the data represents a single day snapshot, end_date should be `None`."
+    )
+    collection_period: tuple[date, date | None] = Field(
+        description="The range of time during which the data was collected. Should be in the format (start_date, end_date)."
+        " If the data represents a single day snapshot, end_date should be `None`."
+    )
+    except_next_update: date | None = Field(
```
Comment (andrewphilipsmith) [inline python/popgetter/metadata.py:78]: This is a typo (not introduced in this PR, but I've just spotted it). It should be `expected_next_update`, not `excepted_next_update`.

File: python/popgetter/assets/us/__init__.py [hunk 2]
```diff
@@ -111,25 +116,48 @@ def generate_variable_dictionary():
...
+                                          potential_denominator_ids:idPath,
+                                          parent_metric_id: idPath[-1]
+                                      })
+            metrics.append(metadata)
+
+    period = 5 if summary_level == 'fiveYear' else 1
+    ACSRelease = SourceDataRelease({
+            name: f"ACS_{year}_{summary_level}",
+            date_published: date(year,1,1),
+            reference_period: [date(year - peroid, 1,1), date(year,31,12 )],
```
Comment (andrewphilipsmith) [inline python/popgetter/assets/us/__init__.py:148]: typo here: `period`, not `peroid`.

**Review by andrewphilipsmith — APPROVED** [review]
Apologies if you were waiting for me to review this!

Mostly this looks good to me, bar a couple of typos. Also, there is a merge conflict with `main` that will need to be resolved.

**Comment by sgreenbury** [comment]
Closing as superseded by #114

**Pull request closed without merging**